app/actions/LedgerActions.js

```diff
@@ -155,11 +158,24 @@ function doWithTransport(fn) {
       return fn(transport).then((r) =>
         transport
           .close()
-          .catch((/*e*/) => {}) // throw?
+          .catch((e) => {
+            throw e;
+          })
```

Comment:
redundant, it will be thrown by default

Reply:
Ah, will keep this in mind and fix at some point.

Reply:
Was mac alright?